scheduling/non_preemptive_shortest_job_first.py

```diff
@@ -0,0 +1,116 @@
+# SJF
+"""
+Non-preemptive Shortest Job First
```

Comment:
I think SJF is non-preemptive itself? So I think this program is similar to https://github.com/TheAlgorithms/Python/blob/master/scheduling/shortest_job_first.py

so we can get rid of `non_preemptive_` from filename

Reply:
The SJF is a non-preemptive scheduling algorithm.
I should name mine without `non_preemptive_`, but there is the same name but different algorithm. (what you mentioned)
The existing program is a preemptive variant of SJF.
So I add `non_preemptive_` to distinguish them.

If we get rid of `non_preemptive_`, I think the other filename(original `shortest_job_first.py`) also should be renamed to `shortest_remaining_time.py`.

Reply:
Ah ok, you may rename `scheduling/shortest_job_first.py` to `scheduling/preemptive_shortest_job_first.py`